BioArchLinux/jasp-desktop/PKGBUILD

```diff
@@ -32,79 +33,98 @@ depends=('r'
-    "r-jaspcircular"
-    "r-jaspcochrane"
-    "r-jaspdistributions"
-    "r-jaspequivalencettests"
-    "r-jaspjags"
-    "r-jasplearnbayes"
-    "r-jasplearnstats"
-    "r-jaspmachinelearning"
-    "r-jaspmetaanalysis"
-    "r-jaspnetwork"
-    "r-jasppower"
-    "r-jaspprocess"
-    "r-jaspprophet"
-    "r-jasppredictiveanalytics"
-    "r-jaspreliability"
-    "r-jasprobustttests"
-    "r-jaspsem"
-    "r-jaspsummarystatistics"
-    "r-jaspsurvival"
-    "r-jasptimeseries"
-    "r-jaspvisualmodeling"
-    "r-jaspacceptancesampling"
-    "r-jaspqualitycontrol"
-    "r-jaspbff"
-    "r-jaspbfpack"
+    librdata
+    libfreexl
 )
 provides=($_pkgname)
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/jasp-stats/jasp-desktop/archive/refs/tags/v${pkgver}.tar.gz"
     'jasp.sh'
-    "jaspColumnEncoder-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspColumnEncoder/archive/refs/tags/v${_ColumnEncoder_ver}.tar.gz"
-    "jaspBase-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspBase/archive/refs/tags/v${pkgver}.tar.gz"
-    "jaspGraphs-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspGraphs/archive/refs/tags/v${pkgver}.tar.gz"
-    "jaspModuleInstaller-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspModuleInstaller/archive/refs/tags/v${pkgver}.tar.gz"
+    "jaspColumnEncoder-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspColumnEncoder/archive/32c53153da95087feb109c0f5f69534ffa3f32b7.zip"
+    "jaspBase-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspBase/archive/0f0f59e18f5524ea422cb3dc5ad76ace6f0518e4.zip"
+    # "jaspGraphs-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspGraphs/archive/refs/tags/v${pkgver}.tar.gz"
+    "jaspModuleBundleManager-${pkgver}.tar.gz::https://github.com/jasp-stats/jaspModuleBundleManager/archive/466057fe2ca0602917be964ce032c4cb03a8619a.zip"
+    "remove_binary_pkgs.patch::https://github.com/jasp-stats/jasp-desktop/commit/3ff41e292c.patch"
 )
-sha256sums=('9078459cc091ac722552058733ef64c5792092c5fe5615d968b515054bc7efe9'
-            'e0714d980e7549b4c7dcbae50370e95b6ad2e7f0cf21a534ceb3a5a83ee583fd'
-            '340de4aa4218fe6216b3a260b75c8256f3b6f501aadcf2c32253aa0e9a2f3c73'
-            '327bc22a5d0ab1a12935bf00327c7075df266a51a1097e502c0b89bc395ba6da'
-            '8db633010ac5ab56d12397dc391f515e7ee4b0faea38ddadac849b5be1d4e241'
-            '4abacceb643e9dcea4f1353ce200292bc395e21766b0998f0898d4cc70cd1ec8')
+sha256sums=('7c0e0fdd6a28ddd484ccd39069da6cb20e03547d2d8ec14b69df5f05cabfa0b2'
+    'e0714d980e7549b4c7dcbae50370e95b6ad2e7f0cf21a534ceb3a5a83ee583fd'
+    '8afdd494d59bc6812fb4d957070d7a92eb21eceecc954636abb264a37e1bcf86'
+    '6a6ed34ed5f2514fff7e9568c045ccf530586ed37ce08e409729c52ba1336c47'
+    'c6e58ca68cc157b7ba56ed6ac83ad1340be5c9f2148c80fb9713562e2dbe6e3f'
+    'f5c145c33cdcb4c254f7e9f0eb7639e8cdf6887e56e2f7c15074973504469889')
+
+_add_libs() {
+    local JASP_LIBS lib
+    JASP_LIBS=(
+        # base
+        "jaspBase"
+        "jaspGraphs"
+        "jaspTools"
```

Comment:
So this is not JASP's Analysis modules, that error messages is only an analysis modules check.